From 62d8c2f5b4818cd2e1ec3dfe22393bb6e5fa7308 Mon Sep 17 00:00:00 2001
From: Wan Shuang <shuang.wan@intel.com>
Date: Thu, 27 Sep 2018 21:05:52 +0800
Subject: [PATCH 2/4] Grant HWC service with SYS_ADMIN capability.

Grants HWC service with SYS_ADMIN privilege to ensure it be able
to grab DRM master role.

Change-Id: I6a053a34ec439dd392e3ab425f0099e49d72dbcd
Tracked-On: None
Signed-off-by: Wan Shuang <shuang.wan@intel.com>
---
 .../default/android.hardware.graphics.composer@2.1-service.rc   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc b/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc
index 5a5b51e8e..473d515a6 100644
--- a/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc
+++ b/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc
@@ -2,6 +2,6 @@ service vendor.hwcomposer-2-1 /vendor/bin/hw/android.hardware.graphics.composer@
     class hal animation
     user system
     group graphics drmrpc
-    capabilities SYS_NICE
+    capabilities SYS_NICE SYS_ADMIN
     onrestart restart surfaceflinger
     writepid /dev/cpuset/system-background/tasks
--
2.21.0
